ADALiOS/ADALiOS/ADAuthenticationContext.m

```diff
@@ -124,6 +124,7 @@ + (void) load
                         [[UIApplication sharedApplication] setDelegate:(__bridge id)CFRetain((__bridge CFTypeRef)appDelegate)];
                     }
                     
+                    SAFE_ARC_RELEASE(observer);
```

Comment:
... this code is all iOS specific, meaning it should be using ARC and protected with ifdefs... so I don't think SAFE_ARC_RELEASE is the right answer here.